index.bs

```diff
@@ -12521,6 +12518,19 @@ values passed to \[[Call]]:
 </ol>
 
 
+<h4 id="preventextensions">Platform object \[[PreventExtensions]] method</h4>
+
+The internal \[[PreventExtensions]] method of every [=platform object=] |O|
```

Comment:
So I define the behavior _only_ for the case where the object supports named/index props. Seems to be more correct and more forward compatible than also defining the behavior in other cases.